rfcs/0051-stop-transaction-processing.md

```diff
@@ -0,0 +1,158 @@
+# Stop processing transactions after a certain slot
+
+This PR describes the feature to stop processing transactions after a certain
+slot, to be used in the Berkeley hard fork.
+
+## Summary
+
+Transactions come from a client or the gossip network and are processed by BPs
+and SNARK workers to be included in blocks.
+
+In this RFC, the procedure to stop processing any new transactions after a
+certain slot is described. This is, any blocks produced after that slot will
+include no transaction at all.
+
+## Motivation
+
+In a hard fork scenario, we want to halt the preceding network and produce a new
+genesis ledger for the succeeding network. This new genesis ledger should be
+produced from a stabilised ledger from the preceding network. This is, we define
+a point in time (slot) where the network continues to operate but with no
+"activity". In detail, after this slot, the network continues to produce blocks
+but without including any transactions, and ensuring any fees paid are 0. This
```

Comment:
What about coinbase? Is coinbase also reduced to 0? If not, the ledger won't stabilise, as BPs will still be receiving tokens for producing blocks…

Reply:
Yes, it is. This is being clarified in the next revision.